Lib/test/test_unittest/test_loader.py

```diff
@@ -1052,6 +1087,20 @@ def test_loadTestsFromNames__module_not_loaded(self):
             if module_name in sys.modules:
                 del sys.modules[module_name]
 
+    # "The specifier should not refer to a test method in a TestCase-derived
+    # subclass that is an abstract base class"
+    def test_loadTestsFromNames__testmethod_in_abc_TestCase(self):
+        m = types.ModuleType('m')
+        class Foo(unittest.TestCase, metaclass=abc.ABCMeta):
+            @abc.abstractmethod
+            def test_1(self): ...
+            def test_2(self): pass
+        m.Foo = Foo
+
+        loader = unittest.TestLoader()
+        with self.assertRaises(TypeError):
+            loader.loadTestsFromNames(['Foo.test_1', 'Foo.test_2'], m)
```

Comment:
Which name does raise this error? Please add a similar check for both individual names: `loader.loadTestsFromNames(['Foo.test_1'], m)`, etc

Reply:
Agreed. Updated the test accordingly then. Thanks.